docs/proposals/working-groups/20221115-managed-kubernetes.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Working Group for Managed Kubernetes in Cluster API
+authors:
+  - "@jackfrancis"
+reviewers:
+    - "@pydctw”"
+    - "@richardcase"
+    - "@vincepri"
+    - "@sbueringer”"
+    - "@fabriziopandini"
+creation-date: 2022-11-15
+last-updated: 2022-11-15
+status: proposed
+see-also:
+  - https://github.com/kubernetes-sigs/cluster-api/issues/7494
+  - https://github.com/kubernetes-sigs/cluster-api/issues/6988
+---
+# Managed Kubernetes in Cluster API Working Group
+
+This document briefly outlines the scope, communication media, and stakeholders for a formal Working Group dedicated to defining a Cluster API-approved Managed Kubernetes solution.
+
+## Scope
+
+The charter for this Working Group will be standardizing Managed Kubernetes solutions across the Cluster API provider ecosystem.
```

Comment:
I think it is worth specifying which deliverables this working group aims to create (in any).
E.g. to implement a proposal to be submitted to the wider community, to define a checklist that can help ensuring consistency across providers etc.

Reply:
I think we can do that, yeah. The specific deliverables may evolve over time, but I think we can reach consensus on a few concrete things we want to solve that justify the creation of this group.